shotover-proxy/tests/cassandra_int_tests/routing.rs

```diff
@@ -0,0 +1,97 @@
+use crate::helpers::cassandra::{run_query, CassandraConnection, CassandraDriver};
+
+pub async fn create_keyspace(connection: &mut CassandraConnection) {
+    let create_ks: &'static str = "CREATE KEYSPACE IF NOT EXISTS test_routing_ks WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };";
+    run_query(connection, create_ks).await;
+}
+
+pub async fn create_table(connection: &mut CassandraConnection) {
+    let create_table_cql =
+        "CREATE TABLE IF NOT EXISTS test_routing_ks.my_test_table (key int PRIMARY KEY, name text);";
+    run_query(connection, create_table_cql).await;
+}
+
+pub async fn test(
+    shotover_contact_points: &str,
+    shotover_port: u16,
+    cassandra_contact_points: &str,
+    cassandra_port: u16,
+) {
+    let mut shotover = CassandraConnection::new(
+        shotover_contact_points,
+        shotover_port,
+        CassandraDriver::Scylla,
+    )
+    .await;
+    shotover
+        .enable_schema_awaiter(
+            &format!(
+                "{}:{}",
+                cassandra_contact_points.split(',').next().unwrap(),
+                cassandra_port
+            ),
+            None,
+        )
+        .await;
+    let mut cassandra = CassandraConnection::new(
+        cassandra_contact_points,
+        cassandra_port,
+        CassandraDriver::Scylla,
+    )
+    .await;
+
+    create_keyspace(&mut shotover).await;
```

Comment:
why do we generate the keyspace through shotover and then the table directly to cassandra?

Reply:
Just a sanity check that both are working as expect.

Reply:
It does mean we dont run the schema_awaiter logic.
But we havent hit any failures for that yet so maybe scylla does internal logic for schema awaiting on DDL queries?